examples/pagedir/src/server/uploadthing/router.ts

```diff
@@ -5,9 +5,7 @@ import { options } from "~/pages/api/auth/[...nextauth]";
 const f = createUploadthing();
 
 export const uploadRouter = {
-  withMdwr: f
-    .fileTypes(["image"])
-    .maxSize("16MB")
+  withMdwr: f(["image"])
```

Comment:
comments explaining what this does (default file size, count) would be nice

Reply:
We should overhaul the examples to be good. After this